app/src/main.c

```diff
@@ -206,6 +207,7 @@ static SDL_bool parse_port(char *optarg, Uint16 *port) {
 
 static SDL_bool parse_args(struct args *args, int argc, char *argv[]) {
     static const struct option long_options[] = {
+        {"always-on-top",no_argument,       NULL, 'a'},
```

Comment:
`a` is used on the `audio` branch for `--forward-audio`, this is not very important, but what do you think about using `-T`(like _top_, `t` is already taken for `--show-touches`) or `A`?

(you could also add a space after the comma and realign the others)